app/assets/javascripts/mumuki_laboratory/application/avatar.js

```diff
@@ -0,0 +1,28 @@
+mumuki.load(function() {
+  let avatar_id = "";
+
+  $("#edit-avatar-icon").on('click', function(){
+    $('#avatar-picker').modal();
+  });
+
+  $("#user-form").on('submit', function(){
+    if (!avatar_id) {
+      const image_url = $('#user-avatar').attr('src');
+      $(this).append('<input type="hidden" name="user[image_url]" value="' + image_url + '"/>');
+    }
+
+    $(this).append('<input type="hidden" name="user[avatar_id]" value="' + avatar_id + '"/>');
+  });
+
+  $('.avatar-item').on('click', function(){
+    $('#user-avatar').attr('src', $(this).attr('src'));
+    $('#avatar-picker').modal('hide');
```

Comment:
New code should always prefix the classes and ids with `mu-` in order to avoid name clashes and to better distinguish between mumuki and third-parties code.

Reply:
Didn't know what was the mu-criteria for the prefixes. Done

Reply:
We tend to use it in any class that is somewhat _public_ and _stable_. Small hacks don't require it, but it is a way of saying that _hey, you can safely customize this class, it is from mumuki to the world_ when you use custom css code in  your organization or runners expose custom UI code.